collections/motoko/dfx.json

```diff
@@ -4,34 +4,26 @@
       "type": "motoko",
       "main": "src/hashmap.mo"
     },
-    "triemap" : {
+    "triemap": {
       "type": "motoko",
       "main": "src/triemap.mo"
     },
-    "rbtree" : {
+    "rbtree": {
       "type": "motoko",
       "main": "src/rbtree.mo"
     },
-    "splay" : {
-      "type": "motoko",
-      "main": "src/splay.mo"
-    },
     "heap": {
       "type": "motoko",
       "main": "src/heap.mo"
     },
-    "btreemap": {
```

Comment:
Removed because they are no in mops?

Reply:
Yes but Byron just uploaded package to `mops` so I'll add it back in a sec